AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -1150,7 +1167,8 @@ - (void)testOpenUrlWithFirstSessionLogUpdate {
 
   // Then
   XCTAssertTrue(result);
-  OCMReject([distributeMock sendFirstSessionUpdateLog]);
+  XCTAssertEqual(invocations, 0);
+  OCMReject([self.distributeInfoTrackerMock updateDistributionGroupId:OCMOCK_ANY]);
```

Comment:
FYI you don't need to repeat this here as the first `OCMReject` in line 1159 continues here. However I like to keep it for readability.